molecule/default/prepare.yml

```diff
@@ -24,7 +24,7 @@
       until: "'running' in systemctl_status.stdout"
       retries: 30
       delay: 5
-      when: ansible_service_mgr == 'systemd'
+      when: ansible_service_mgr == "systemd"
       changed_when: "'running' in systemctl_status.stdout"
```

Comment:
I don't understand why quotes are necessary here, when they're unnecessary in `handlers/main.yml`.  What is (are) the complaint(s) you see from the linter(s)?

Reply:
```
yamllint.................................................................Failed
- hook id: yamllint
- exit code: 1

molecule/default/prepare.yml
  24:24     error    syntax error: expected <block end>, but found '<scalar>' (syntax)
  ```
  I think the issue may be that the condition in `handlers/main.yml` is a number, while here it is a string

Reply:
I still don't understand, but OK.